File: app/src/main/java/za/co/woolworths/financial/services/android/ui/activities/product/dynamicyield/response/request/Device.java
package za.co.woolworths.financial.services.android.ui.activities.product.dynamicyield.response.request;

import java.io.Serializable;

public class Device implements Serializable {
    private String ip;

    public Device(String ip) {
        this.ip = ip;
    }

    public String getIp() {
        return this.ip;
    }

    public void setIp(String ip) {
        this.ip = ip;
    }
}

File: app/src/main/java/za/co/woolworths/financial/services/android/ui/activities/dashboard/DynamicYield/request/Device.kt
package za.co.woolworths.financial.services.android.ui.activities.dashboard.DynamicYield.request

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Device(
    val ip: String? = null,
    val userAgent: String? = null
): Parcelable